Data/OJS.Data/AdminDbContext.cs

```diff
@@ -23,7 +24,8 @@ protected override void OnModelCreating(ModelBuilder builder)
     private static void ConfigureDateModels(ModelBuilder builder) =>
         builder.Model.GetEntityTypes()
             .SelectMany(entityType => entityType.ClrType.GetProperties())
-            .Where(property => property.GetCustomAttributes<UtcConvertableAttribute>().Any())
+            .Where(property => property.Name is GlobalConstants.Properties.CreatedOn or GlobalConstants.Properties.DeletedOn
```

Comment:
Use `nameOf(IDeletableEntity.DeletedOn)` and `IAuditInfoEntity.CreatedOn`

Reply:
Done.